mongorestore/mongorestore_test.go

```diff
@@ -69,6 +126,9 @@ func TestMongorestore(t *testing.T) {
 	}
 
 	Convey("With a test MongoRestore", t, func() {
+		// As specified in TOOLS-2363, the --db and --collection options
```

Comment:
Do we still need the comment here?

Reply:
We need the comment, but it seems to be in the wrong location. I'll move it to the test case that verifies this behavior so that it's clearer what that's regarding.